components/widgets/datetime/multiple-timezones.js

```diff
@@ -0,0 +1,84 @@
+import { Component } from 'react'
+import styled from 'styled-components'
+import Widget from '../../widget'
+import moment from 'moment-timezone'
+import Table from '../../table'
+import { object, string, array, number, boolean } from 'yup'
+
+const TimeItem = styled.div`
+  font-size: 3em;
+  text-align: center;
+  text-align: right;
+`
+
+const schema = object().shape({
+  format24: boolean(),
+  timeZones: array(object({
+    name: string().required(),
+    flag: string().required()
+  })).required().max(5),
+  interval: number()
+})
+
+export default class MultipleTimezones extends Component {
+  static defaultProps = {
+    interval: 1000 * 10,
+    timeZones: [{ name: 'Asia/Colombo', flag: 'India.png' }, { name: 'Europe/Copenhagen', flag: 'Denmark.png' }],
+    format24: false
+  }
+
+  state = {
+    date: new Date(),
+    error: false,
+    loading: true
+  }
+
+  componentDidMount () {
+    const { interval } = this.props
+
+    schema.validate(this.props).then(() =>
+      this.setState({ loading: false }),
+    this.timeout = setTimeout(() => this.setState({ date: new Date() }), interval))
```

Comment:
The line indentation looks incorrect. Brackets should make it more readable. 😉

Reply:
Should be fixed now.